apps/core/src/utils/calculateAPY.ts

```diff
@@ -22,18 +22,19 @@ export function calculateAPY(
     if (stakingPoolActivationEpoch || stakingPoolActivationEpoch === 0) {
         const numEpochsParticipated = epoch - stakingPoolActivationEpoch;
         apy =
-            Math.pow(
+            (Math.pow(
                 1 +
                     (+stakingPoolSuiBalance - +poolTokenBalance) /
                         +poolTokenBalance,
                 365 / numEpochsParticipated
-            ) - 1;
+            ) -
+                1) *
+            100;
     } else {
         apy = 0;
     }
 
-    //guard against NaN
-    const apyReturn = apy ? roundFloat(apy, roundDecimals) : 0;
+    const apyReturn = roundFloat(apy, roundDecimals);
```

Comment:
You don't need to nan guard anymore?

Reply:
I thought it wasnt needed, I guess it could happen if both `poolTokenBalance` and the difference between `stakingPoolSuiBalance` and `poolTokenBalance` are 0,